<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicPlayer</class>
 <widget class="QWidget" name="MusicPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="OpenBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QSlider" name="ProcessSlider">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="VolumeValueSlider">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>22</width>
     <height>131</height>
    </rect>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="VolumeBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>音量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NextMusicBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>35</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Nex</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StartBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PreMusicBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>35</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Pre</string>
   </property>
  </widget>
  <widget class="QLabel" name="InfoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
